assets/css/main.css

```diff
@@ -4,6 +4,10 @@
 /*
 /*****************************************************************************/
 
+html {
+  scroll-padding-top: 3rem;
```

Comment:
I don't think this does anything
<img width="497" alt="image" src="https://github.com/FarsetLabs/farsetlabs.github.io/assets/635903/91b886ec-04fc-4f0f-aea7-792d1aef24f1">

Reply:
🤔 Is that message a GitHub thing? The `Room Hire` menu options link to the anchors on the Room Hire page, but they're hidden under the fixed nav bar when you get there.

![Screenshot 2024-05-28 at 11 22 45](https://github.com/FarsetLabs/farsetlabs.github.io/assets/25768210/46a86a67-f65b-4894-b729-74ae0814f073)

But that CSS rule moves them down:
![Screenshot 2024-05-28 at 11 24 07](https://github.com/FarsetLabs/farsetlabs.github.io/assets/25768210/7e9732dd-e003-47d9-80fa-6adf50c99c41)

It _is_ possible the browser is accounting for my stupidity here, but I _think_ it's okay that the parent doesn't scroll here? Because I don't care about scroll snapping or anything like that?

Reply:
@robertwpaul That message is in the Firefox dev tools. TIL, that works a treat! ❤️